library/drone.py

```diff
@@ -0,0 +1,103 @@
+"""
+Copyright MIT and Harvey Mudd College
+MIT License
+Summer 2020
+
+Defines the interface of the Drone module of the racecar_core library.
+"""
+
+import abc
+import copy
+import numpy as np
+from nptyping import NDArray
+
+
+class Drone(abc.ABC):
+    """
+    Returns the color image captured by the drone's camera.
+    """
+
+    # The dimensions of the image in pixels
+    _WIDTH: int = 640
+    _HEIGHT: int = 480
+
+
+    def get_drone_image(self) -> NDArray[(480, 640, 3), np.uint8]:

```

Comment:
To keep in line with my comment in the Simulation PR review, consider changing this to `get_image` instead. Happy to discuss if you disagree.